api/api.go

```diff
@@ -331,6 +331,7 @@ func (api *API) listAlerts(w http.ResponseWriter, r *http.Request) {
 	alerts := api.alerts.GetPending()
 	defer alerts.Close()
 
+	api.mtx.RLock()
```

Comment:
What are we locking against here?

Reply:
there's a `routes := api.route.Match(a.Labels)` a few lines below, in a loop. This is one of the fields that gets updated in `Update()`

Reply:
Got it. I missed that.

Reply:
no worries, github didn't bother showing it 🤷‍♂️

Reply:
I actually expanded but missed the `api.route` part. So it's just me needing coffee.